packages/main/src/tray-menu.ts

```diff
@@ -120,6 +120,8 @@ export class TrayMenu {
       this.updateMenu();
     });
 
+    tray.on('double-click', this.showMainWindow.bind(this));
```

Comment:
it looks to me that it's only a Windows need, I'm not aware on apps doing that on macOS

so probably need to add a condition for Windows only

Also, please add a unit test in `tray-menu.spec.ts`

Reply:
The author of #4500 also mentions MacOS, though I cannot confirm that, as I don't own a Mac. But the Electron docs mention the double click also working on MacOS, so I assume it must exist.

I don't think a special condition is needed, as per the Electron docs, the double click event can only be triggered on the supported platforms.

Of course I can write a unit test. I just didn't deem it necessary after looking through the code, as the other click triggers have them neither.